config.example.toml

```diff
@@ -105,6 +105,7 @@
 [brew]
 # greedy_cask = true
 # autoremove = true
+# fetch_head = true
```

Comment:
It would be great if this entry needs a comment stating its functionality, the previous 2 entries don't have documents either, but don't worry, they will be covered in [#636](https://github.com/topgrade-rs/topgrade/pull/636#issuecomment-1879915175)